crypto/bls/signature_batch.go

```diff
@@ -119,12 +172,14 @@ func (s *SignatureBatch) AggregateBatch() (*SignatureBatch, error) {
 			currBatch.Signatures = append(currBatch.Signatures, s.Signatures[i])
 			currBatch.Messages = append(currBatch.Messages, s.Messages[i])
 			currBatch.PublicKeys = append(currBatch.PublicKeys, s.PublicKeys[i])
+			currBatch.Descriptions = append(currBatch.Descriptions, s.Descriptions[i])
```

Comment:
I have added length validation for descriptions. Thanks for the comment!

I reordered the empty check and length check because I think length mismatch of signatures and messages should  always be an error even in case that signatures are empty and messages are non-empty.